data/data/aws/vpc/vpc-private.tf

```diff
@@ -24,6 +24,21 @@ resource "aws_route" "to_nat_gw" {
   }
 }
 
+# We can't target the NAT gw for our "private" IPv6 subnet.  Instead, we target the internet gateway,
+# since we want our private IPv6 addresses to be able to talk out to the internet, too.
```

Comment:
TODO(@abhinavdahiya): need to get clarity on this.

Reply:
What kind of clarity are you looking for?  The AWS NAT gateway doesn't support IPv6.  They hand out globally routable IPv6 addresses and expect those to be used.

Reply:
https://github.com/openshift/enhancements/pull/87#discussion_r354945664

Reply:
I've looked into doing what we discussed on the enhancement (switching to an egress only internet gateway).  A major downside is that tags are not supported.  A second potential downside is that we may end up wanting some direct connectivity to Nodes to exercise more of IPv6 with ingress, since AWS load balancers can only do IPv4.  At this point I'd suggest we leave this as-is ...

https://docs.aws.amazon.com/cli/latest/reference/ec2/create-egress-only-internet-gateway.html

https://docs.aws.amazon.com/cli/latest/reference/ec2/create-egress-only-internet-gateway.html